src/patternfly/components/Button/examples/Button.md

```diff
@@ -383,6 +392,7 @@ Semantic buttons and links are important for usability as well as accessibility.
 | `.pf-m-block` | `.pf-c-button` | Creates a block level button. |
 | `.pf-m-control` | `.pf-c-button` | Modifies for control styles. **Note:** This modifier should only be used when using buttons in the Input Group or Clipboard Copy components. |
 | `.pf-m-expanded` | `.pf-c-button.pf-m-control` | Modifies a control button for the expanded state. |
+| `.pf-m-selected` | `.pf-c-button` | Modifies a control button for the selected state. |
```

Comment:
@mcarrano @mmenestr @mceledonia is there a guideline for when to use "selected" instead of "expanded"?

Reply:
Idk of a guideline - will defer to @mcarrano on that one, but I want to say that "selected" makes more sense in this case because it's a technically just a button that's been selected, regardless of what it leads to.